app/models/exceptions/not_implemented.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Exceptions
+  class NotImplemented < RuntimeError
+    attr_reader :message
+
+    def initialize message
+      @message = message
+    end
```

Comment:
The body of this `class` is superfluous: it's just re-implementing the default behavior of `RuntimeError`: 
```
[1] pry(main)> e = RuntimeError.new('test')
=> #<RuntimeError: test>
[2] pry(main)> e.message
=> "test"
```